### PR TITLE
made czech translation less akward

### PR DESCRIPTION
**Pull request by CerxMe**
changed wording to make it sound more natural, added new translations

### CODE REVIEW

File: translations/base-cz.yaml [hunk 1]
```diff
@@ -483,7 +482,7 @@ storyRewards:
     # Those are the rewards gained from completing the store
     reward_cutter_and_trash:
         title: Řezání tvarů
-        desc: Právě jste odemknuli <strong>řezač</strong> - řeže tvary <strong>svisle</strong> bez ohledu na svou orientaci!<br><br>Nezapomeňte se zbavovat odpadu, jinak <strong>se vám zasekne produkce</strong> - pro tento účel jsem vám odemknul koš, který můžete použít na mazání odpadu!
+        desc: Právě jste odemknuli <strong>pila</strong> - řeže tvary <strong>svisle</strong> bez ohledu na svou orientaci!<br><br>Nezapomeňte se zbavovat odpadu, jinak <strong>se vám zasekne produkce</strong> - pro tento účel jsem vám odemknul koš, který můžete použít na mazání odpadu!
```
Comment (greeny) [inline translations/base-cz.yaml:485]: `pilu`

File: translations/base-cz.yaml [hunk 2]
```diff
@@ -218,27 +217,27 @@ dialogs:
...
         title: Potvrdit smazání
         desc: >-
-            Odstraňujete spoustu budov (přesněji <count>)! Opravdu to chcete udělat?
+            Odstraňujete spoustu budov (přesněji <count>)! Opravdu je chcete smazat?
 
     blueprintsNotUnlocked:
         title: Zatím neodemčeno
         desc: >-
-            Plány ještě nebyly odemčeny! Chcete-li je odemknout, dokončete úroveň 12.
+            Dokončete level 12 pro odemknutí plánů.
```
Comment (greeny) [inline translations/base-cz.yaml:233]: "not yet unlocked" got lost here?

File: translations/base-cz.yaml [hunk 3]
```diff
@@ -186,11 +185,11 @@ dialogs:
 
     resetKeybindingsConfirmation:
         title: Reset klávesových zkratek
-        desc: Vaše klávesové zkratky budou resetovány. Prosím potvrďte.
+        desc: Opravdu chcete vrátit klávesové zkratky zpět do továrního nastavení?
```
Comment (greeny) [inline translations/base-cz.yaml:188]: this means "reset to factory settings" @tobspr is that OK? (there's no factory technically, I'd go with "default settings" - `původní nastavení`)

File: translations/base-cz.yaml [hunk 4]
```diff
@@ -54,7 +53,7 @@ global:
     error: Chyba
 
     # How big numbers are rendered, e.g. "10,000"
-    thousandsDivider: " "
+    thousandsDivider: ","
```
Comment (greeny) [inline translations/base-cz.yaml:56]: Nope, cz numbers use space, not comma (it's thousands divider)

File: translations/base-cz.yaml [hunk 5]
```diff
@@ -186,11 +185,11 @@ dialogs:
 
     resetKeybindingsConfirmation:
         title: Reset klávesových zkratek
-        desc: Vaše klávesové zkratky budou resetovány. Prosím potvrďte.
+        desc: Opravdu chcete vrátit klávesové zkratky zpět do továrního nastavení?
 
     keybindingsResetOk:
         title: Reset klávesových zkratek
-        desc: Vaše klávesové zkratky byly resetovány do původního nastavení!
+        desc: Vaše klávesové zkratky byly resetovány do továrního nastavení!
```
Comment (greeny) [inline translations/base-cz.yaml:192]: Same as above

File: translations/base-cz.yaml [hunk 6]
```diff
@@ -218,27 +217,27 @@ dialogs:
...
-            Plány ještě nebyly odemčeny! Chcete-li je odemknout, dokončete úroveň 12.
+            Dokončete level 12 pro odemknutí plánů.
 
     keybindingsIntroduction:
         title: Užitečné klávesové zkratky
         desc: >-
             Hra má spoustu klávesových zkratek, které usnadňují stavbu velkých továren.
             Zde jsou některé, ale nezapomeňte se podívat i na <strong>ostatní klávesové zkratky</strong>!<br><br>
-            <code class='keybinding'>CTRL</code> + Táhnout: Vybrání oblasti.<br>
+            <code class='keybinding'>CTRL</code> + Táhnout: Vyberte oblast pro zkopírování / smazání.<br>
```
Comment (greeny) [inline translations/base-cz.yaml:240]: the string was changed to "Drag to select an area", needs to be reflected here

File: translations/base-cz.yaml (hunk 3 above)
Comment (tobspr) [reply translations/base-cz.yaml:188]: Both are fine for me :)

**Comment by tobspr** [comment]
See annotations from @greeny

File: translations/base-cz.yaml (hunk 3 above)
Comment (CerxMe) [reply translations/base-cz.yaml:188]: I went with "factory settings" just makes it abundantly clear you're about to reset back to default, but I guess `původní` would work as well.

**Comment by greeny** [comment]
Looks good to me

**Comment by tobspr** [comment]
Thanks a lot!

**Pull request merged**